bazaarvoice/emodb
# PD-223162-BLOB-Client-giving-EmoDbClientException

## Github Issue #

[1234](https://github.com/bazaarvoice/emodb/issues/11)

## What Are We Doing Here?

Capturing the emodb Exception and classifying them into more granule level exception, so the msg could be more clearer to the clinets of emoDB

## How to Test and Verify
      Tested locally.
<img width="1496" alt="image" src="https://github.com/bazaarvoice/emodb/assets/120096920/7cb07e9f-0034-4067-a240-5e809af47356">


1. Check out this PR
2. Run Command X, Click Button Y
3. Profit

## Risk
 low

### Required Testing

`Smoke`, `Regression`, or `Manual`. (All changes except documentation need smoke
testing at a minimum).

### Risk Summary

Add one or a few complete sentences about the possible risks or concerns for
this change.

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
